### PR TITLE
PB-6000 API expose for adding labels to namespace

### PR DESCRIPTION
**Pull request by prabhu-px**
**What this PR does / why we need it**:
Exposes adding labels to namespace

**Which issue(s) this PR fixes** (optional)
Closes # PB-6000

**Special notes for your reviewer**:
```
API: taas/namespaces/addLabel
method: post
body: {
    "namespaces": ["central","ns1"],
    "ns_label": {"env":"dev"}
}

}

### CODE REVIEW

**Comment by prabhu-px** [comment]
image

File: apiServer/taas/utils/torpedoutils.go [hunk 1]
```diff
@@ -420,3 +421,37 @@ func GetVMsWithNamespaceLabels(c *gin.Context) {
...
+	if err := c.BindJSON(&NamespaceLabelRequest); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+	if len(NamespaceLabelRequest.Label) == 0 {
+		c.JSON(http.StatusBadRequest, gin.H{"error": "namespace labels cannot be empty"})
+		return
+	}
+	log.Infof("NamespaceLabelRequest", NamespaceLabelRequest)
+	_, err := pdslib.UpdatePDSNamespce(NamespaceLabelRequest.Namespace, NamespaceLabelRequest.Label)
```
Comment (mkoppal-px) [inline apiServer/taas/utils/torpedoutils.go:447]: Please use
```
Inst().S.AddNamespaceLabel(NamespaceLabelRequest.Namespace,  NamespaceLabelRequest.Label)
```
This function is directly in `k8s.go`

**Comment by prabhu-px** [comment]
request modified:
```
{
    "namespaces": ["central","ns1"],
    "ns_label": {"env":"dev"}
}

**Comment by prabhu-px** [comment]
image

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**